base/io.jl

```diff
@@ -658,13 +658,13 @@ Commit all currently buffered writes to the given stream.
 flush(io::IO) = nothing
 
 """
-    skipchars(io::IO, predicate; linecomment=nothing)
+    skipchars(io::IO, predicate; linecomment::Char)
```

Comment:
Yea, and the function signature is not restricted to `Char` so this should just be restored.